src/service/viewer-impl.js

```diff
@@ -1005,7 +1005,7 @@ export class Viewer {
       return Promise.reject(getChannelError());
     }
     return this.messagingReadyPromise_.then(() => {
-      return this.sendMessageUnreliable_(eventType, data, awaitResponse);
+      return this.messageDeliverer_(eventType, data, awaitResponse);
```

Comment:
Got you. but logically it doesn't make too much sense because the messaging is there, and we know we can send it "reliablely", why still call "sendUnreliable". Anyway, I don't want to be off the topic here too much since we're actually thinking of comletely removing `sendMessageUnreliable_`,